install-panel.sh

```diff
@@ -874,6 +845,22 @@ function main {
     [ -z "$MYSQL_PASSWORD" ] && print_error "MySQL password cannot be empty"
   done
 
+  print_brake 88
+  echo "* Please follow the steps below. The installer will ask you for configuration details."
+  print_brake 88
+  echo ""
+
+  echo "* Choose your timezone. e.g. [Europe/Amsterdam]: " # This is really hard without autocomplete
+  read -r timezone_input
+
+  [ -z "$timezone_input" ] && timezone="Europe/Amsterdam" || timezone=$timezone_input
+
+  echo "* Provide the email address that will be used to configure LetsEncrypt and Egg author: " # Make it more readable 
```

Comment:
```suggestion
  echo "* Provide the email address that will be used to configure Let's Encrypt and Pterodactyl: "
```

I am aware the `Egg author` is a more correct and specific term, but perhaps it's okay to just say it's an email that Pterodactyl uses?